Added ⚡️ content to to `Onboarding` >  `Funding a wallet` #2

Bosch-0 opened the pull request:
⚡️[Live preview](https://deploy-preview-590--sad-borg-390916.netlify.app/guide/onboarding/funding-a-wallet/)

Added Lightning content to the `Funding a wallet` page in `Onboarding`. Big thanks for @pavlenex for the feedback / guidance! 

Closed https://github.com/BitcoinDesign/Guide/pull/569 in favor of this. Made a lot of changes based on feedback on that PR as well as going over the request page made me re-think a lot of the content here. 

This page focuses on the funding flow of a standard non-custodial mobile Lightning wallet that does not use any Lightning services. I have a sub-page called 'Lightning services' for this page that is almost complete and will be up once this PR is merged (as feedback here will effect that page). That page will cover the same funding flow but using Lightning services. 

Lightning services obviously give the superior UX, should that be the primary application we are designing for? Wanted to discuss this further with you all. For now I think it's good to have both wallet types laid out (with and without LN services) until there is easy plug-and-play Lightning services tools around (Dunder / Blocktank are making strides here). 

Also moved this page to be after `First use` in this chapter closing https://github.com/BitcoinDesign/Guide/issues/550 

Closes https://github.com/BitcoinDesign/Guide/issues/518

Bosch-0 commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,300 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
- - No exchange fees
- - Simplifies onboarding
- {% include fact/close.html %}
+The first step in having users fund your wallet is finding out if they currently own bitcoin or not. This will determine what kind of information to present to your users next as well as guide them towards how they will fund their wallet. If the user wants to fund their wallet at a later time they should have the option to skip this process and jump straight into the wallet.
```
I think we should cover that in `Restoring a wallet` and focus on this page being an extension of `First use`

Bosch-0 commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,300 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+   alt-text = "Screen guiding user where to buy bitcoin."
+   width = 250
+   height = 541
+%}
```
It's too hard to recommend places unless the wallet is only located in one location. But agree we could have a better solution here. 

Personally though I think Googling would be the number one way people do find exchanges to buy bitcoin though despite the scams (I don't think it's that rampant of an issue).

GBKS replied:
This is a situation where a crowdsourced community user guide would be helpful. Like a design guide, but it would focus on how-to for users. Wallets could link out to it for various things like backup tutorials, how-to-buy bitcoin, etc. Doesn't make much sense to me that all wallets need to recreate this stuff. Not sure how much of this already exists in one form or another.

pavlenex replied:
I think that a wallet optionally integrating a way for users to buy btc can be a good revenue stream for them as well as a smooth UX.

sbddesign replied:
I agree with @pavlenex.

- If the user is inexperienced enough to where they need to be told to find bitcoin via a Google search, then they are inexperienced enough to need to be guarded from accidentally stumbling into a scam or accidentally acquiring Bitcoin Cash or Doge.
- If the user is experienced enough to be trusted finding bitcoin via a Google search, then they are so experienced they don't need to be told where to find bitcoin.

You do raise a good point @Bosch-0 about exchanges not working in every jurisdiction. It would be a very US-centric worldview to simply declare "just use XYZ Exchange". Nonetheless, I think the risk of accidentally sending the user down a Google rabbit hole is worse.

One potential solution: wallet has an (entirely optional) search function where the user can seek out an exchange. "Select your country" to find a recommended exchange.

Bosch-0 replied:
What about just making this generic and mentioning it's up to the application to recommend where to send users to buy?

GBKS reviewed (changes requested):
This has become quite a complex page with the Lightning addition. I think you have all the right pieces here, but I find that the writing could be polished a bit further, and the page could flow better.

I left a good amount of feedback to simplify sentences. While reading, I tried to pay attention to smoothly I could parse the content, and there were various sentences I had to reread. That's where I left the comments.

Note that you used `will` 14 times and `should` 17 times. I think you can avoid most of them and be more direct. 

I think this page could benefit from a flow chart, or you could consider moving the overview at the bottom to the top of the page. I found it difficult to follow the flow through the written text. Creating that mental map of the user flow at the top could help readers fill in the details as they read.

Another idea, how about using our Fact/Tip/Recommendation box style for pointing out the benefits of Lightning services? That way it's super easy to identify those pointers.

sbddesign commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,300 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+
+If the user has purchased some bitcoin either outside or within your application, or they already own some, the next thing should be explaining how their wallet works once its funded. 
+
+Once users funds their wallet and they have a channel open they will initially only be able to send Lightning payments. This is due to the payment channel the user opens in later steps only having outbound liquidity. This can also be an opportunity to give a soft introduction to what a payment channel is and how they work. This should be clear to users before they fund the wallet to avoid future  ...
```
@GBKS The whole "How it works" section confused me a lot on my first read-through. I felt the same way @GBKS; I wasn't clear if the channel opening had already occurred or was going to occur in the future.

Having read the page multiple times, I think @Bosch-0 is trying to convey a scenario similar to Blixt where you fund your wallet on-chain and then a channel is automatically opened once your funds are cleared on-chain. Thus, it's most proper to explain to the user what is going to happen to their money.

So I see the logic in needing to explain to the reader what is going to happen in a later step. Perhaps it could be more concise or worded differently.

Bosch-0 replied:
Yes @sbddesign is correct with what I was aiming to do here, I will make this section more concise as to avoid confusion.

sbddesign commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,300 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+
+### Network sync
+
+Depending on how your application connects to the Bitcoin network you may want to show the status of the network sync on the same screen as the funding payment request. 
```
Hardline view: users should not be allowed to generate an on-chain address until block filters have synced for the first time.

I have witnessed a problem with wallets where you send money to the wallet, and then don't see it for several hours due to block filter sync issues. That's fine for me personally; I grasp the concept of "still syncing" and "go check a block explorer if in doubt". I just fear that's too much for onboarding a new user.

Thoughts, anyone? Is my viewpoint too radical?

sbddesign commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,300 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+
+<div class="image-slide-gallery">
+
+{% include picture.html
```
I am surprised that "View on block explorer" is a very large call-to-action button.

If the wallet has its own block filters, I don't see why the wallet can't display its own transaction confirmation status.

If someone new to bitcoin clicks that giant CTA, they are going to end up on a block explorer showing them TXIDs, on-chain addresses, fees per vbytes, and all sorts of stuff that will look like complete gibberish to them.

"View in block explorer" is an option that's available but tucked away for an advanced users, IMHO.

sbddesign commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,300 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
-{% include fact/pros.html %}
-  - An easy and popular way to acquire bitcoin
-{% include fact/close.html %}
+## Overview
```
@Bosch-0 Yeah, I agree. I think if you put the entire flow at the top of the page, it would make more sense. Just showing the reader the entire flow before discussing it point-by-point would be helpful.

For example, per the [discussion in this thread](https://github.com/BitcoinDesign/Guide/pull/590/files#r768507191), the "How it works" section might make more sense if the reader has already seen an overview of the whole flow.

sbddesign reviewed (changes requested):
Thanks for all the hard work on this page @Bosch-0. I spent a **lot** of time reading this page. I think the initial opening of channels is turning out to be one of the most challenging topics in Lightning.

- There are a couple of typographic and grammatical suggestions
- I think moving the completed user flow closer to the top will help the reader
- I encourage you to look at the example user flow and consider what can be _removed_ from each frame.
  - Does a new user need a block explorer link?
  - Does a new user need to know what a Lightning node is?
  - Should a user be allowed to send/receive when initial block filter sync is still happening?
  - These are all my opinions; they could be wrong but I raise them for consideration.

Thinking ahead to the future: I think we should try mapping out a similar funding flow with different types of LSP scenarios. Separate PRs, perhaps.

Bosch-0 commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,300 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+<div class="image-slide-gallery">
 
-<div class="center" markdown="1">
+{% include picture.html
```
That was intentional, just a preference of mine but will add the tilt for consistency with other UI kit images :)

Yes also intentional, that is a detour from the main on-boarding screens. Will be re-working this page based on above discussion and will make this more clear also.

Bosch-0 commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,300 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+   height = 541
+%}
+
+{% include picture.html
```
Only if you are using Lightning services this can be done. Or if the user manually opens an inbound channel with say LNBig

Bosch-0 commented:
@sbddesign @pavlenex @GBKS 

Hey guys, I addressed mostly all your feedback with the latest commit. Let me know what you think of the changes :)

I removed the `How it works` section as that seemed a common point of confusion. It's probably better to explain these things when the user gets to the stage where they are relevant to understand.

GBKS commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
-{% include image.html
+The first step in funding a wallet is asking the user if they currently own bitcoin.
+
+If your user does not own bitcoin your application should guide them in obtaining some. One solution is offering a guide the user could read on how to obtain bitcoin.
```
```suggestion
If your user does not own bitcoin, your application should guide them in obtaining some. One solution is offering a guide the user could read on how to obtain bitcoin.
```

GBKS commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+
+If your user does not own bitcoin your application should guide them in obtaining some. One solution is offering a guide the user could read on how to obtain bitcoin.
+
+Alternatively, you could offer the option to buy directly within your application which greatly simplifies the funding process.
```
```suggestion
Alternatively, you could offer the option to buy directly within your application, which greatly simplifies the funding process.
```

GBKS commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+
+Alternatively, you could offer the option to buy directly within your application which greatly simplifies the funding process.
+
+If the user wants to fund their wallet at a later time they should have the option to skip this funding process.
```
```suggestion
If the user wants to fund their wallet at a later time, they should have the option to skip this funding process.
```

GBKS commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+   height = 541
+%}
+
+{% include picture.html
```
This screen is very busy and lacks a clear call-to-action for newcomers. I'd put up a big "Send your bitcoin to the address below" title. I'd also remove the completeness indicator, makes me think that I can't send the bitcoin yet. Also think the note about the channel and the "What is a channel?" link should be right next to each other.

Bosch-0 replied:
Made some changes, what do you think? 

I think letting the user know that their funds won't appear until that sync is complete is very important. I re-worded it so it's clear you can still fund while it is in progress. 

Having the user think their funds have gone missing is a bigger problem to avoid than thinking they can not fund straight away. Though as mentioned above you can explain that they can still fund while the sync is in progress with some copy.

GBKS replied:
Quick mock-up for simplifying this.

<img width="1050" alt="Frame 2949" src="https://user-images.githubusercontent.com/695901/147095287-3e94b3a5-9b84-4b1f-b3e5-63c6dd623ab0.png">

pavlenex replied:
@GBKS @Bosch-0 Should we update the mock or is this something we can want to do in a follow-up? Would be cool if we can get this baby merged if everything else looks good. I like the simplified version.

Bosch-0 replied:
I think this misses quite a lot of important things - if a channel is being automatically opened the user will have less funds available once that happens. If it is not explained that a channel will be opened I'd imagine they would be confused as to why they received less. I like the way the network sync is presented though. I'd have to change quite a few things if I used the above image so maybe add these in a follow up? idm 

Can you share the Figma file for this @GBKS

pavlenex replied:
Yup, just asked @GBKS and we agreed this is better suited for a follow-up and some explorations. We can even invite people to post their own solutions in a workshop kind of. format after providing a case for it. Just an idea, but can be just a simple follow up PR. 

Let's merge this baby 🚀

GBKS replied:
I think that channel opening info + fee should be explained in a separate page that comes before this invoice screen. It's just too much for a single screen.

GBKS commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+   image = "/assets/images/guide/onboarding/funding-a-wallet/own-bitcoin.png"
+   retina = "/assets/images/guide/onboarding/funding-a-wallet/own-bitcoin@2x.png"
+   layout = "shadow"
+   caption = "Ask the user if they own bitcoin."
```
Do you need the two dots? I assume the paths are different based on what choice you make, so you might not know how many steps are still to come.

GBKS commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+   height = 541
+%}
+
+{% include picture.html
```
Would be cool to have a custom icon here that reflects what is happening.

GBKS commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+Below is an overview of a standard wallet funding flow. This expands on what we covered in the previous page covering the user's [first use]({{'/guide/onboarding/first-use/' | relative_url}}).
 
-<div class="center" markdown="1">
+## Does the user own bitcoin?
```
Could put numbers in the titles to indicate the steps?

GBKS commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
-{% include fact/close.html %}
+### Network sync
 
+If your application uses a method like compact block filters (BIP157/158) to sync to the Bitcoin network, which may take some time to sync, communicate to the user that their funds will not appear until the sync is complete. You may even consider not presenting a payment request until this sync is complete to prevent user confusion that their funds have not shown up.
```
```suggestion
Sync methods like compact block filters (BIP157/158) to sync to the Bitcoin network take some time to finish. If your application uses them, you should communicate to the user that their funds will not appear until the sync is complete. You may even consider not presenting a payment request until this sync is complete. This can help prevent users from getting confused about their funds not showing up.
```
Tried to shorten sentences.

GBKS commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+## Funding the wallet
 
-### Redeem bitcoin using a gift card
+The next step for the user to take is sending bitcoin to the payment request.
```
```suggestion
The next step for the user is to send bitcoin to the payment request.
```

GBKS commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+The next step for the user to take is sending bitcoin to the payment request.
 
-Gift cards allow users to buy and own small amounts of bitcoin without having to sign up for an exchange. These cards are sometimes available from local retailers.
+Once the bitcoin has been sent, indicate to the user when the wallet has received the transaction and communicate it's status. Once the initial transaction has been confirmed, it's recommended to open a payment channel for the user automatically. This avoids the user having to manually find a peer and open a channel.
```
```suggestion
Once the bitcoin has been sent, indicate to the user when the wallet has received the transaction and communicate it's status. Once the initial transaction has been confirmed, it's recommended to open a payment channel for the user automatically. This way, the user does not have to manually find a peer and open a channel.
```

GBKS commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+Once the bitcoin has been sent, indicate to the user when the wallet has received the transaction and communicate it's status. Once the initial transaction has been confirmed, it's recommended to open a payment channel for the user automatically. This avoids the user having to manually find a peer and open a channel.
 
-Users can get set up in minutes using this technique.
+Opening of a payment channel will require another on-chain transaction to be processed. If you choose to automatically open a channel for the user both the initial and channel opening transaction status can be indicated with one screen. This avoids exposing users to unnecessary complexity.
```
```suggestion
Opening of a payment channel will require another on-chain transaction to be processed. If you choose to automatically open a channel for the user, the status of both the initial and the channel opening transactions can be indicated with one screen. This avoids exposing users to unnecessary complexity.
```
Tried to make the sentence easier to read.

Don't quite understand the last bit about unnecessary complexity.

Bosch-0 replied:
> Don't quite understand the last bit about unnecessary complexity.

That is referring to showing the status of both the on-chain transactions. I'd imagine it would be confusing for a user if they were exposed to two payments being processed with the one payment they send.

GBKS commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
-Users can get set up in minutes using this technique.
+Opening of a payment channel will require another on-chain transaction to be processed. If you choose to automatically open a channel for the user both the initial and channel opening transaction status can be indicated with one screen. This avoids exposing users to unnecessary complexity.
+
+Once a payment channel is open, the user's wallet is ready to use. As the user's newly opened channel only has outbound liquidity they will not be able to receive Lightning payments straight away.
```
```suggestion
Once a payment channel is open, the user's wallet is ready to use. As the newly opened channel only has outbound liquidity, they will not be able to receive Lightning payments yet.
```

GBKS commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+
+{% include /tip/open.html label="⚡ Lightning services" icon="info" color="blue" %}
+
+When it comes to funding a wallet, Lightning services can prevent users waiting for confirmations to use their wallet as. They can also enable channels with inbound liqudity to be opened to the user so they can receive Lightning payments straight away.
```
Don't understand the first sentence.

GBKS commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+
+</div>
+
+{% include /tip/open.html label="⚡ Lightning services" icon="info" color="blue" %}
```
Could those tip titles include the name of the service, if the service has one?

GBKS commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+
+### Manual channel opening
+
+If a user wants to manually open a channel they should have the option to disable automatic channel opens in the settings. This is a more involved process and should be focused on more experienced users and not be the primary user flow.
```
```suggestion
If a user wants to manually open a channel they should have the option to disable automatic channel opens in the settings. This is a more involved process and should be focused on more experienced users, and not be the primary user flow.
```

pavlenex commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+Once the bitcoin has been sent, indicate to the user when the wallet has received the transaction and communicate it's status. Once the initial transaction has been confirmed, it's recommended to open a payment channel for the user automatically. This avoids the user having to manually find a peer and open a channel.
 
-Users can get set up in minutes using this technique.
+Opening of a payment channel will require another on-chain transaction to be processed. If you choose to automatically open a channel for the user both the initial and channel opening transaction status can be indicated with one screen. This avoids exposing users to unnecessary complexity.
```
> Opening of a payment channel will require another on-chain transaction to be processed.

How about zero-confirmation channels? So instead of `will require` I'd put `usually requires, unless you choose to take a bit more risk and offer zero-cons..`

Bosch-0 replied:
That will be covered in the Lightning services sub-page. However, there is a tip at the end of the `Funding a wallet` section that hints towards this.

pavlenex commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
-Users can get set up in minutes using this technique.
+Opening of a payment channel will require another on-chain transaction to be processed. If you choose to automatically open a channel for the user both the initial and channel opening transaction status can be indicated with one screen. This avoids exposing users to unnecessary complexity.
+
+Once a payment channel is open, the user's wallet is ready to use. As the user's newly opened channel only has outbound liquidity they will not be able to receive Lightning payments straight away.
```
> As the user's newly opened channel only has outbound liquidity they will not be able to receive Lightning payments straight away.

I think this is a pain-point we can try to offer solution to. For manual channel opening this is okay to happen, but if an LSP opens channel to us?

Breez resolves this by opening % on top of the amount, but again this is risk a wallet provider may or may not be wiling to make.

Bosch-0 replied:
So I aimed to cover using LSPs in a sub-page of the funding a wallet page whereas this flow uses none. I added the tips to point towards a solution for this (these will have links to the sub-page once it is live).

pavlenex commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+
+<div class="center" markdown="1">
+
+{% include image.html
```
![Screenshot 2021-12-20 at 10 10 33](https://user-images.githubusercontent.com/36959754/146751783-c06efd30-6937-4d24-97d7-cf032eceeac7.png)

pavlenex reviewed (commented):
I really like the flow-distinction and the direction in which this PR is heading. I have a few suggestions bellow 🎸

Bosch-0 commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+{% include /tip/close.html %}
 
-### Buying bitcoin
+Below is an overview of a standard wallet funding flow. This expands on what we covered in the previous page covering the user's [first use]({{'/guide/onboarding/first-use/' | relative_url}}).
```
Changed copy a bit here :)

Bosch-0 commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+
+## The funding request
+
+The next step is present to the user an on-chain payment request so they can fund their wallet.
```
Changed to just using payment request

Bosch-0 commented:
Addressed everyone's comments, requesting another overview / ACKs :) @GBKS @sbddesign @pavlenex

pavlenex commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,281 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+
+It's recommended to start any required syncing as soon as the user opens your application to limit wait times.
+
+## 3. Sending the bitcoin
```
```suggestion
## 3. Sending bitcoin
```

pavlenex replied:
Also, calling this `sending bitcoin` is a bit weird, from end-user perspective it's the "receiving", right? Unless. you are trying to emphasise self-sending here but I don't think we should cover that.

Bosch-0 replied:
Well it's the action the user taking to fund their wallet. What do you mean by self-sending? The user is sending it to them selves from whatever external source they have their bitcoin

pavlenex replied:
Yes, that's exactly what I mean, I just think calling it sending may sound confusing. Perhaps initial funding, but I am not really sure, my perception of it is that it's weird, perhaps other's don't think it is.

pavlenex commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,281 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+
+The next step for the user is to send bitcoin to the wallet.
+
+Once the bitcoin has been sent, notify the user once it's received. This should be followed by notifying the user that their wallet is being setup. The setup process being the confirmation of the first transaction and opening of a new payment channel.
```
```suggestion
Once bitcoin has been sent, notify the user once it's received. This should be followed by notifying the user that their wallet is being setup. The setup process being the confirmation of the first transaction and opening of a new payment channel.
```

pavlenex commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,281 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
-{% include fact/pros.html %}
-  - An easy and popular way to acquire bitcoin
-{% include fact/close.html %}
+For beginner users, it's recommended to open a payment channel for them automatically once the initial transaction is confirmed. This way, they don't have to manually find a peer and open a channel.
```
Who are they opening channel to? Is a wallet provider opening it to them, or is user opening to someone automatically. I think we should be clear here on `who is who` so people don't get lost, I did get lost a bit here tbh.

And if a wallet provider is opening a channel on behalf of users, isn't that then a lightning service? Confused since you specifically said this flow excludes any lightning service.

Bosch-0 replied:
To the wallet providers Lighting node automatically yes. They aren't opening a channel for the user the application just automatically opens to a pre-selected node (the wallets ideally). So I wouldn't call this a Lightning service per se. Will clarify this some more

pavlenex replied:
So what you're suggesting here is just auto-channel opening to a random node? I think that's really bad practice, but if it opens channels to the provider's node only, that to me is a service, because it won't work unless provider provides certain % on top of the liquidity like Breez does.

Bosch-0 replied:
No I'm not recommending that, it would be to the wallet providers node like Blixt does. 

You can have a channel opened to the wallet providers node without any interaction from them, service, or additional fees. It would only have outbound capacity in this scenario. 

It's just creating and broadcasting a channel open automatically without the user manually doing the process.

pavlenex replied:
I see now what you mean, can we then revise this so it's a bit clearer and make sure to tell about the tradeoffs like having only onbound capacity?

pavlenex commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,281 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+
+{% include /tip/open.html label="⚡ Pay-to-open" icon="info" color="blue" %}
+
+Offering a pay-to-open Lightning service allows users to receive Lighting payments as soon as they fund their wallet.
```
This is great, but on top of the page we said "no lightning service"
![Screenshot 2021-12-20 at 18 14 29](https://user-images.githubusercontent.com/36959754/146807142-3b600336-3f13-47f9-9c50-c09191dc0519.png)
, so I am confused a bit 😄

Bosch-0 replied:
See my comment above about auto-channel opens. I don't consider that to be a Lightning service it's something that the app can just be automated to do.

pavlenex replied:
I am not sure, to me this still sounds like a type of "service" we're doing on behalf of the user.

pavlenex commented on guide/onboarding/funding-a-wallet.md:
```diff
@@ -22,101 +30,281 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
+
+## 3. Sending the bitcoin
+
+The next step for the user is to send bitcoin to the wallet.
+
+Once the bitcoin has been sent, notify the user once it's received. This should be followed by notifying the user that their wallet is being setup. The setup process being the confirmation of the first transaction and opening of a new payment channel.
```
This again complicates the flow and emphasises self-sending?

Bosch-0 replied:
I'm not sure what you mean by self-sending?

pavlenex replied:
> Once the bitcoin has been sent, notify the user once it's received.

I think whoever is reading this is lost at this point on what's happening, who's doing what.

pavlenex reviewed (commented):
Good improvements, I re-read the page top to bottom again, and discovered some inconsistencies, and was a bit confused with the `Send bitcoin` section.

Some of these are simply question and come from my misunderstanding of the desired flow perhaps 😄

Bosch-0 commented:
Made some more updates based on feedback, requesting (hopefully) a final review

pavlenex reviewed (commented):
@Bosch-0 LGTM, just a ⚡ Pay-to-open info box is duplicated twice in text, not sure if that's intentional or not. Also you should resolve image conflicts so we can merge it in.

I think this page is in a shape where we can get it merge, but it'll likely need some tweaking once all pages from the pending PR's are there. We'll also need to get some hands on feedback from people to see if things are clear enough, since it touches quite a few new(technical) concepts.

Bosch-0 commented:
Some more changes added, re-worked a few things which I think has made it more clear on the automatic channel open part - still need to discuss this further though we should follow that up in another PR. 

Fixed conflict also

GBKS reviewed (approved):
ACK. I'd like to review some of the UX choices in our Payments follow-up task to ensure consistency across the pages, and also update the UI kit to include all the best practices we've outlined.
